spec/features/peeps_spec.rb

```diff
@@ -0,0 +1,25 @@
+feature 'Peeps' do
+  scenario 'be able to send peeps' do
+    register
+    sign_in
+    peep
+    expect(page).to have_content("Hello world")
```

Comment:
"Hello world" is a magic string - why are you expecting the page to have "hello world"? If your helpers accepted arguments, this would be more readable. For instance `peep("Hello World")` would make it really clear that your expectation is looking for the content of the Peep.